app/src/main/res/layout/activity_bill_detail.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:bind="http://schemas.android.com/apk/res-auto"
+    xmlns:car_view="http://schemas.android.com/tools"
+    >
+
+    <data>
+
+        <import type="com.framgia.fsalon.utils.Utils"/>
+
+        <import type="com.framgia.fsalon.utils.LayoutManager"/>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.fsalon.screen.billdetail.BillDetailViewModel"
+            />
+
+    </data>
+
+    <android.support.v4.widget.NestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            >
+
+            <ProgressBar
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerInParent="true"
+                android:visibility="@{viewModel.progressBarVisibility}"
+                />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_below="@+id/toolbar"
```

Comment:
WARNING: [UnknownIdInLayout] The id "`toolbar`" is not referring to any views in this layout
 The `@+id/` syntax refers to an existing id, or creates a new one if it has not already been defined elsewhere. However, this means that if you have a typo in your reference, or if the referred view no longer exists, you do not get a warning since the id will be created on demand.

This is sometimes intentional, for example where you are referring to a view which is provided in a different layout via an include. However, it is usually an accident where you have a typo or you have renamed a view without updating all the references to it.